config/config_spineGeHemis.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
this file should now be moved under ivadomed/config, see https://github.com/neuropoly/ivado-medical-imaging/pull/197